docs/pages/getting-started/providers/rso.mdx

```diff
@@ -0,0 +1,141 @@
+
+## Setup
+
+### Callback URL
+
+<Code>
+  <Code.Next>
+
+```bash
+https://example.com/api/auth/callback/rso
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```bash
+https://example.com/auth/callback/rso
+```
+
+  </Code.Svelte>
+</Code>
+
+### Environment Variables
+
+```
+AUTH_RSO_ID
+AUTH_RSO_SECRET
+```
+
+### Configuration
+
+<Code>
+  <Code.Next>
+
+```ts filename="/auth.ts"
+import NextAuth from "next-auth"
+import RSO from "next-auth/providers/rso"
+
+export const { handlers, auth, signIn, signOut } = NextAuth({
+  providers: [RSO],
+})
+```
+
+  </Code.Next>
+  <Code.Svelte>
+
+```ts filename="/src/auth.ts"
+import { SvelteKitAuth } from "@auth/sveltekit"
+import RSO from "@auth/sveltekit/providers/rso"
+
+export const { handle, signIn, signOut } = SvelteKitAuth({
+  providers: [RSO],
+})
+```
+
+  </Code.Svelte>
+  <Code.Express>
+
+```ts filename="/src/app.ts"
+import { ExpressAuth } from "@auth/express"
+import RSO from "@auth/express/providers/rso"
+
+app.use("/auth/*", ExpressAuth({ providers: [RSO] }))
+```
+
+  </Code.Express>
+</Code>
+
+#### How to get RSO Access
+
+- The be granted RSO access, the first step is to register an **Production** App in the
+  developer portal `https://developer.riotgames.com/`. Before requesting a production app
+  make sure you read `General Policies` and `Tournament Policies` carefully
+  (it's a popup when trying to create an app) and prepare yourself with it. Another note
+  - requesting RSO access with the intent of "testing" it out will result in a deny.
+- After the Application has been created and reviewed/approved by one of the developers
+  you will need to create another app in the beta developer portal.
+  Prepare yourself with the following information in order to complete the creation:
+  - Contact Emails
+  - Name of the company/service (Client Name)
+    - Will also appear on the RSO Login Page
+  - Logo of the company/service
+    - Should scale between **275x275** and **60x60**
+    - For best results, they recommend using a PNG file
+    - The logo will be displayed on both light and dark backgrounds
+  - Privacy policy URL
+  - Terms of service URL
+  - Redirect URLs (can be changed after creation (but not to localhost as explained below))
+    - A list of URLs you would like the RSO Client to redirect the players after login
+    - Typically the base callback url for AuthJS - NextJS: https://example.com/api/auth/callback/rso
+  - Post Logout Redirect URLs
+    - https://example.com/api/auth/signout
+
+In `.env` create the following entries:
+
+```
+RSO_CLIENT_ID=<copy Application (client) ID here>
+RSO_CLIENT_SECRET=<copy generated client secret value here>
+```
+
+<Callout type="warning">
+  Riot Sign On currently does not allow local development so you need to have a
+  public domain to test it or create a proxy in your environment.
+</Callout>
+
+### Signed-JWT Method
```

Comment:
In this section, maybe we just take that callout warning out of the callout, i.e. just put that as regular text under this heading. And then remove the superfluous example at the bottom. It's not adding anything new that the above examples aren't, right? 

Also it uses the pages router path and doesn't use the env var inference, so it could be confusing to new users

Reply:
True that, moved/removed it now.